week4/dhKim/src/components/Content.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+const Content = ({ items }) => {
+    return (
+        <div>
+        <div className="product_container">
+            <article className="product_bottombanner_container">
+                {items.filter((item)=>item.isSold)
+                .map(
+                    (
+                        { title, image, location, timeAgo, price, comments, likes, isSold },index
+                    ) => (
+                        <section className="item1" key={index}>
+                            <div className="product_image">
+                                <img src={image} alt={title} />
+                            </div>
+                            <div className="product_info_container">
+                                <ul className="product_info">
+                                    <li className="product_name">{title}</li>
+                                    <li className="place_buy_container">
+                                        <span className="place_buy">{location}</span>
+                                        <span>·</span>
+                                        <span className="date_buy">{timeAgo}</span>
+                                    </li>
+                                    <li className="price">{price}</li>
+                                </ul>
+                                {/* 좋아요나 커맨트가 있는 것들만 표시되도록.. */}
+                            <div className="comment_emogi_container">
+                                <div className="heart">
+                                    <img src="/assets/heart.svg" alt="좋아요" />
+                                    <span className="heart_num">{likes}</span>
+                                </div>
```

Comment:
```suggestion
                                {heart && (<div className="heart">
                                    <img src="/assets/heart.svg" alt="좋아요" />
                                    <span className="heart_num">{likes}</span>
                                </div>)}
```
아니면 더 명시적으로 heart !== 0 && (view) 이렇게 할 수도 있어요!